components/homme/src/share/cxx/scream_kokkos_utils.hpp

```diff
@@ -0,0 +1,189 @@
+#ifndef SCREAM_KOKKOS_UTILS_HPP
+#define SCREAM_KOKKOS_UTILS_HPP
+
+#include "Types.hpp"
+#include "ExecSpaceDefs.hpp"
+
+#include "../../algorithms/src/Kokkos_Random.hpp"
```

Comment:
This is misleading at best. I think it works cause it switches to a system include, and uses the search path `${KOKKOS_SRC}/core/src`. I think the best way to handle this is to modify the include directory in `cmake/Kokkos.cmake`, and add also `${KOKKOS_SRC}/algorithms/src`.

Reply:
I don't like this either, but I'm not eager to dive into Homme/Kokkos cmake integration.

Reply:
I can take care of that, if you want.